opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -98,7 +98,7 @@
     private static String configFilename = null;
     private static int status = 0;
 
-    private static final ArrayList<String> repositories = new ArrayList<>();
+    private static final Set<String> repositories = new HashSet<>();
```

Comment:
the trick here just to do the dedup? or is there any other magic?

Reply:
yes, this is just to avoid duplicities.